coursefunctions.h

```diff
@@ -189,47 +189,47 @@ void TestDrawTriangle(Buffer2D<PIXEL> & target)
         verts[1] = {150, 452, 1, 1};
         verts[2] = {50, 452, 1, 1};
         PIXEL colors1[3] = {0xffff0000, 0xffff0000, 0xffff0000};
-        // Your color code goes here for 'attr'
+        attr[0].color = colors1[0];//
```

Comment:
This is good for now, but next we will need to have all three points of color go into the attr[] 3 indices